src/test/java/firstStep/SetTest.java

```diff
@@ -0,0 +1,54 @@
+package firstStep;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
```

Comment:
```suggestion
    void setUp() {
```

컨벤션은 꼭꼭 반드시 확인해 주세요!

> - 자바 코드 컨벤션을 지키면서 프로그래밍한다.
>   - 이 과정의 Code Style은 [intellij idea Code Style. Java](https://www.jetbrains.com/help/idea/code-style-java.html)을 따른다.
>   - [intellij idea Code Style. Java](https://www.jetbrains.com/help/idea/code-style-java.html)을 따르려면 code formatting 단축키(Windows : Ctrl + Alt + L. Mac : ⌥ (Option) + ⌘ (Command) + L.)를 사용한다.